plugins/module_utils/network/vyos/argspec/firewall_rules/firewall_rules.py

```diff
@@ -190,7 +190,14 @@ def __init__(self, **kwargs):
                                 "recent": {
                                     "options": {
                                         "count": {"type": "int"},
-                                        "time": {"type": "int"},
+                                        "time": {
+                                            "type": {
+                                                "choices": [
+                                                    "second",
+                                                    "minute",
+                                                    "hour"
+                                                ]
```

Comment:
You're missing a close brace here.
```suggestion
                                                ]
                                            }
```